docs/docs/cmd/spo/file/file-roleinheritance-break.md

```diff
@@ -0,0 +1,54 @@
+# spo file roleinheritance break
+
+Breaks inheritance of file. Keeping existing permissions is the default behavior.
+
+## Usage
+
+```sh
+m365 spo file roleinheritance break [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site where the item for which to break role inheritance is located
```

Comment:
Personally I find this description a bit too verbose. We shouldn't repeat what this command is doing in every description 😊

```suggestion
: URL of the site where the file is located
```